mods/webui/src/stories/generictoggle/GenericToggle.styles.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2024 by Fonoster Inc (https://fonoster.com)
+ * http://github.com/fonoster/fonoster
+ *
+ * This file is part of Fonoster
+ *
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Switch, { SwitchProps } from "@mui/material/Switch";
+import { styled } from "@mui/material/styles";
+
+export const StyledMuiSwitch = styled(Switch)(({ theme }) => ({
+  width: 50,
```

Comment:
Would you review the dimensions of the component? Figma shows a width of 32x16 px for the overall component and 22x22 px for the inner circle. 

The component should be much smaller than the current implementation. For example, using the code produced by Figma Dev Mode and plugging the values in React Playground, you can see the difference in size:

Directly from Figma:

<img width="1508" alt="Screenshot 2024-12-09 at 8 15 08 PM" src="https://github.com/user-attachments/assets/0f3861ba-e6fc-4360-94d0-0339d5f4f0ea">

Values from current implementation:

<img width="1510" alt="Screenshot 2024-12-09 at 8 18 18 PM" src="https://github.com/user-attachments/assets/54a020a5-de97-4e43-9a11-c23cdeaf6213">

Reply:
okay